Automattic/wp-calypso
# Staging Sites: Improve staging site card design

* Closes https://github.com/Automattic/dotcom-forge/issues/1930

## Proposed Changes

* This PR improves the design of a staging card by adding a site icon and a site URL. 

## Testing Instructions

* Make sure you have a staging site created (if not, create one)
* Set a custom icon on a **staging site** under `/settings/general/` along with the site title
* Navigate to `hosting-config/{site}` of the parent site
* Check if the site icon and site title of a staging site are present on the staging card:

<img width="1255" alt="Screenshot 2023-04-04 at 10 22 19 AM" src="https://user-images.githubusercontent.com/25575134/229732481-8ab379a4-f9b9-4a74-8b1f-4075ab7191a8.png">

* Check the view from mobile (using Chrome dev tools):

<img width="573" alt="Screenshot 2023-04-04 at 10 23 46 AM" src="https://user-images.githubusercontent.com/25575134/229732730-849fa51b-32a0-4d3a-8d11-03b00b33a817.png">

## Pre-merge Checklist
